### PR TITLE
base management script

### PR DESCRIPTION
**Pull request by pnuckowski**

### CODE REVIEW

File: alamo_worker/manage.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        )
+
+        self.config = ConfigParser()
+        self.config.read(config_file)
+
+    def execute(self):
+        self.parse_config()
+
+        zero_mq_port = self.config.get('zero_mq', 'remote_host')
+        zero_mq_host = self.config.get('zero_mq', 'remote_port')
```
Comment (h00dy) [inline alamo_worker/manage.py]: mq_host ... 'remote_port'

File: alamo_worker/manage.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            'config.cfg'
+        )
+
+        self.config = ConfigParser()
+        self.config.read(config_file)
+
+    def execute(self):
+        self.parse_config()
+
+        zero_mq_port = self.config.get('zero_mq', 'remote_host')
```
Comment (h00dy) [inline alamo_worker/manage.py]: mq_port ... 'remote_host'

**Comment by ch40s3k** [comment]
+1

**Comment by deejay1** [comment]
:+1:

**Pull request merged**